libr/debug/p/native/linux/linux_debug.c

```diff
@@ -97,6 +97,68 @@ int linux_handle_signals(RDebug *dbg, int tid) {
 		// siginfo.si_code -> HWBKPT, USER, KERNEL or WHAT
 		// TODO: DO MORE RDEBUGREASON HERE
 		switch (dbg->reason.signum) {
+		case SIGABRT: // 6 / SIGIOT // SIGABRT
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGALRM:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGBUS:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGCHLD:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;;
+			break;
+		case SIGCONT:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+#if __sparc__
+		case SIGEMT:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+#endif
+		case SIGFPE:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGHUP:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGILL:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGPWR:
+			break;
+		case SIGINT:
+			dbg->reason.type = R_DEBUG_REASON_USERSUSP;
```

Comment:
i dont understand the meaning of USERUSP. SIGINT is what happens when the user presses ^C. so SIGINT or REASON_USERINTERRUPT or REASON_INTERRUPT seems more meaningful